hoomd/BoxResizeUpdater.h

```diff
@@ -14,51 +14,84 @@
 
 #include "Updater.h"
 #include "Variant.h"
+#include "BoxDim.h"
 
 #include <memory>
+#include <string>
+#include <stdexcept>
 #include <pybind11/pybind11.h>
 
 #ifndef __BOXRESIZEUPDATER_H__
 #define __BOXRESIZEUPDATER_H__
 
-//! Updates the simulation box over time
-/*! This simple updater gets the box lengths from specified variants and sets those box sizes
-    over time. As an option, particles can be rescaled with the box lengths or left where they are.
-
-    \ingroup updaters
+/// Updates the simulation box over time
+/** This simple updater gets the box lengths from specified variants and sets
+ * those box sizes over time. As an option, particles can be rescaled with the
+ * box lengths or left where they are.
```

Comment:
Not exactly related to this PR, but I would add a warning here indicating the limitation discussed in [this issue](https://github.com/glotzerlab/hoomd-blue/issues/568) (`scale_particles` doesn't work correctly in MPI simulations).